Ginger/GingerCoreNET/Reports/GingerExecutionReport.cs

```diff
@@ -2704,6 +2709,11 @@ public static string CreateGingerExecutionReport(ReportInfo RI, bool calledFromA
             switch (RI.reportInfoLevel)
             {
                 case ReportInfo.ReportInfoLevel.RunSetLevel:
+                    int totalRunners = WorkSpace.Instance.RunsetExecutor.Runners.Count;
```

Comment:
Put all this inside 1 function and add try-catch. Because with current code if any exception then an entire report will come as blank.